### PR TITLE
VI-SPDAT CE Assessments

### PR DESCRIPTION
**Pull request by awisema**

### CODE REVIEW

File: drivers/vispdats/app/models/vispdats/synthetic/base.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    end
+
+    def assessment_level
+      # FIXME
+      1 # Crisis Needs Assessment
+    end
+
+    def priortization_status
+      # FIXME
+      2 # Not placed on prioritization list
```
Comment (eanders) [inline drivers/vispdats/app/models/vispdats/synthetic/base.rb]: We should check, but this should probably be based on VI-SPDAT score.

File: drivers/vispdats/app/models/vispdats/synthetic/base.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    def assessment_type
+      # FIXME
+      3 # In Person
+    end
+
+    def assessment_level
+      # FIXME
+      1 # Crisis Needs Assessment
+    end
```
Comment (eanders) [inline drivers/vispdats/app/models/vispdats/synthetic/base.rb]: These are probably fine.

File: drivers/vispdats/app/models/vispdats/synthetic/base.rb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      new_vispdats = GrdaWarehouse::Vispdat::Base.where.not(id: self.select(:source_id))
+      new_vispdats.find_each do |vispdat|
+        next unless vispdat.client.present? && vispdat.submitted_at.present?
+
+        create(enrollment: find_enrollment(vispdat), client: vispdat.client, source: vispdat)
+      end
+    end
+
+    def self.find_enrollment(vispdat)
+      project_candidates = GrdaWarehouse::Hud::Project.pluck(:ProjectID) # FIXME
```
Comment (eanders) [inline drivers/vispdats/app/models/vispdats/synthetic/base.rb]: Should we limit this by project type, or a flag on projects that are eligible?

File: drivers/vispdats/app/models/vispdats/synthetic/base.rb [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      source.submitted_at&.to_date
+    end
+
+    def assessment_location
+      source.user&.agency&.name
+    end
+
+    def assessment_type
+      # TODO add to the VI-SPDAT
+      3 # In Person
```
Comment (eanders) [inline drivers/vispdats/app/models/vispdats/synthetic/base.rb]: Isn't this what you just added with contact_method?

**Review by eanders — APPROVED** [review]

File: drivers/vispdats/app/models/vispdats/synthetic/base.rb (hunk 4 above)
Comment (awisema) [reply drivers/vispdats/app/models/vispdats/synthetic/base.rb]: Good catch! Added.

**Pull request merged**